### PR TITLE
Fixing the admonition and updating it as a step with admonition

### PR DESCRIPTION
**Pull request by Srivaralakshmi**
**Aligned team**: Dev Tools

**Purpose**: To resolve the following issues:
(https://issues.redhat.com/browse/RHDEVDOCS-5537)

**OCP version this PR applies to**: enterprise-`4.10`,`4.11`,`4.12`, `4.13`, and `4.14`

**Link to docs preview**: 
- [Logging in to the Argo CD instance by using the Argo CD admin account](https://63364--docspreview.netlify.app/openshift-enterprise/latest/cicd/gitops/installing-openshift-gitops#logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account_installing-openshift-gitops)

**SME review**: Completed by @reginapizza 
**QE review**: Completed by @varshab1210
**Peer-review**: Completed by @deerskindoll

### CODE REVIEW

**Comment by Srivaralakshmi** [comment]
@reginapizza and @varshab1210: PTAL at this PR - https://github.com/openshift/openshift-docs/pull/63364 and provide feedback. Thank you!

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc [hunk 1]
```diff
@@ -17,18 +17,18 @@
 
 . In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators* to verify that the {gitops-title} Operator is installed.
 . Navigate to the {rh-app-icon} menu -> *OpenShift GitOps* -> *Cluster Argo CD*. The login page of the Argo CD UI is displayed in a new window.
-. Obtain the password for the Argo CD instance:
+. Optional: To log in with your {product-title} credentials, ensure you are a user of the `cluster-admins` group and then select the `LOG IN VIA OPENSHIFT` option in the Argo CD user interface.
++
+[NOTE]
+====
```
Comment (Srivaralakshmi) [inline modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:23]: @reginapizza and @varshab1210 : Should we say `oc adm groups new cluster-admins admin` or just `oc adm groups new cluster-admins` ?

Comment (reginapizza) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:23]: cluster-admins I think

**Review by reginaelyse — APPROVED** [review]

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc (hunk 1 above)
Comment (Srivaralakshmi) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:23]: @varshab1210 Can you please verify and confirm?

Comment (varshab1210) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:23]: **$ oc adm groups new cluster-admins** creates a **cluster-admins** group but with **$ oc adm groups new cluster-admins admin** it also creates an **admin** user

Comment (varshab1210) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:23]: Hi @Srivaralakshmi it should something like `$ oc adm groups new cluster-admins <user>`

**Comment by Srivaralakshmi** [comment]
@varshab1210 I have revised the PR as per your suggestion. PTAL and approve. Thanks!

**Review by varshab1210 — APPROVED** [review]

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc [hunk 2]
```diff
@@ -17,18 +17,18 @@
...
+To be a user of the `cluster-admins` group, use the `oc adm groups new cluster-admins <user>` command, where `<user>` is the default cluster role that you can bind to users and groups cluster-wide or locally.
```
Comment (deerskindoll) [inline modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:24]: I'd start this sentence with "use the `oc adm groups new cluster-admins <user>` command" to make it more action-oriented and reword the rest accordingly.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc [hunk 3]
```diff
@@ -17,18 +17,18 @@
 
 . In the *Administrator* perspective of the web console, navigate to *Operators* -> *Installed Operators* to verify that the {gitops-title} Operator is installed.
 . Navigate to the {rh-app-icon} menu -> *OpenShift GitOps* -> *Cluster Argo CD*. The login page of the Argo CD UI is displayed in a new window.
-. Obtain the password for the Argo CD instance:
+. Optional: To log in with your {product-title} credentials, ensure you are a user of the `cluster-admins` group and then select the `LOG IN VIA OPENSHIFT` option in the Argo CD user interface.
```
Comment (deerskindoll) [inline modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:20]: I'd start the sentence with "ensure/make sure you are a user of the `cluster-admins` group" to make this more action-oriented and reword the rest accordingly.

**Review by deerskindoll — COMMENTED** [review]
2 nitpicks, otherwise lgtm

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc (hunk 2 above)
Comment (Srivaralakshmi) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:24]: As per the IVM style guide: If you orient the user to the location where the action is performed **or explain the purpose of the action,** do so before describing the action.

@deerskindoll Retaining as is.

File: modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc (hunk 3 above)
Comment (Srivaralakshmi) [reply modules/logging-in-to-the-argo-cd-instance-by-using-the-argo-cd-admin-account.adoc:20]: As per the IVM style guide: If you orient the user to the location where the action is performed **or explain the purpose of the action,** do so before describing the action.

@deerskindoll Retaining as is.

**Review by deerskindoll — APPROVED** [review]
lgtm

**Pull request merged**